OBSDOCS-1980 add note on developer perspective in 4.19+ (#98299)
openshift/openshift-docs: observability/cluster_observability_operator/ui_plugins/monitoring-ui-plugin.adoc

:FeatureName: The {coo-full} monitoring UI plugin
include::snippets/technology-preview.adoc[leveloffset=+2]

include::snippets/unified-perspective-web-console.adoc[]

> The monitoring UI plugin adds monitoring features to the OpenShift web console.

Reviewer: Similarly to the previous comment about usage of "OpenShift":
```suggestion
The monitoring UI plugin adds monitoring features to the {product-title} web console.
```

* **{rh-rhacm}:** The monitoring plugin in {coo-first} allows it to function in {rh-rhacm-first} environments, providing {rh-rhacm} with the same alerting capabilities as {product-title}. You can configure the plugin to fetch alerts from the {rh-rhacm} Alertmanager backend. This enables seamless integration and user experience by aligning {rh-rhacm} and {product-title} monitoring workflows.

* **Incident detection:** The incident detection feature groups related alerts into incidents, to help you identify the root causes of alert bursts, instead of being overwhelmed by individual alerts. It presents a timeline of incidents, color-coded by severity, and you can drill down into the individual alerts within an incident. The system also categorizes alerts by affected component, grouped by severity. This helps you focus on the most critical areas first.
+

> The incident detection feature is available in the OpenShift web console at **Observe** → **Incidents**.

Reviewer: Similarly to the previous comment about usage of "OpenShift", plus small adjustments:
```suggestion
The incident detection feature is available in the {product-title} web console on the  *Observe* -> *Incidents* page.
```

> include::modules/coo-monitoring-ui-plugin-install.adoc[leveloffset=+1]

Reviewer: the "Using Cluster Observability Operator incident detection" module still contains the Administrator perspective mention, see in the preview: https://98299--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/cluster_observability_operator/ui_plugins/monitoring-ui-plugin#coo-incident-detection-using_monitoring-ui-plugin
> In the Administrator perspective of the web console, click on Observe → Incidents.
